image/templates/helm/stackrox-central/templates/01-central-13-deployment.yaml.htpl

```diff
@@ -142,6 +142,10 @@ spec:
           value: "true"
         {{- end }}
         [<- end >]
+        [< if .FeatureFlags.ROX_POLICY_AS_CODE >]
```

Comment:
This feature flag has no meaning if we don't check for it in the config controller - we should.

Reply:
One way to describe what this is doing in plain English: "If the policy-as-code feature flag is enabled when the Helm template is generated, add the `ROX_POLICY_AS_CODE` flag to the Central deployment, with the value being read from the `configAsCode.enabled` field in the values file.

The thinking is that the env var shouldn't be added at all if the Helm chart wasn't generated with policy-as-code enabled since there's no way the config-controller would ever get deployed (the config-controller template files are excluded).  That said, I admit it's kind of confusing that the same env var (`ROX_POLICY_AS_CODE`) is used at two different levels here.